src/Util/UrlBuilder.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace QuizApp\Util;
+
+/**
+ * Class UrlBuilder
+ * @package QuizApp\Util
+ */
+class UrlBuilder
+{
+
+    /**
+     * UrlBuilder constructor.
+     */
+    public function __construct()
+    {
+    }
+
+    public function getUrl(array $queryParameters, int $page = 1): string
+    {
+        $resultString = '?';
+        if ($page !== 1) {
+            $resultString .= "page=$page&";
+        }
```

Comment:
In which case could the $value be the empty string and why?

Reply:
The UrlBuilder shouldn't be in this pr, it was a mistake I made using git stash on 2 different branches